_posts/en/newsletters/2021-01-20-newsletter.md

```diff
@@ -0,0 +1,125 @@
+---
+title: 'Bitcoin Optech Newsletter #132'
+permalink: /en/newsletters/2021/01/20/
+name: 2021-01-20-newsletter
+slug: 2021-01-20-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes posts to the Bitcoin-Dev mailing list
+about payjoin adoption and making hardware wallets compatible with more
+advanced Bitcoin features.  Also included are our regular sections with
+overviews of changes to services and client software, new releases and
+release candidates, and changes to popular Bitcoin infrastructure
+software.
+
+## News
+
+- **Payjoin adoption:** Chris Belcher [posted][belcher payjoin] to the
+  Bitcoin-Dev mailing list a request for people to look for ways to
+  increase [payjoin][topic payjoin] adoption along with a [wiki
+  page][payjoin wiki] tracking the projects that provide either sending
+  or receiving support for payjoin.  One [suggestion][raw payjoin] by
+  Craig Raw was to extend the protocol to allow it to work even when the
+  receiver doesn't operate a server.
+
+- **Making hardware wallets compatible with more advanced Bitcoin features:**
+  Kevin Loaec [started a discussion][loaec hww] on the Bitcoin-Dev
+  mailing list about how hardware wallets could be changed to allow them
+  to handle scripts more complicated than single-sig or multisig.  For
+  example, allowing a hardware wallet to handle in-channel LN payments
+  or payments made from a [vault][topic vaults].  His post does a good
+  job of describing various problems that current hardware wallets can't
+  handle, but he notes that necessary "changes may be very difficult".
```

Comment:
```suggestion
  handle, but he notes that necessary "changes may be very difficult."
```

Reply:
We discussed the order of quotation mark and fullstop on a previous newsletter, but I don't remember whether there was a general ruling on it. In German, punctuation only goes inside the quotation marks when it belongs to the quoted sequence, e.g. when it's a question or a whole sentence is part of the quote. When the punctuation belongs to the carrier sentence, it belongs outside of the quotation marks. I think this may also differ between British English and American English. I would have considered it correct before this suggestion.

Reply:
My feedback is based on grammar correctness in English; I'm not aware of a difference between American and British usage here, but could be wrong.

Reply:
Given that @harding writes in American English though, the quote before the period would have received red ink corrections on any paper I submitted in the US 😄

Reply:
I don't have a preference.  Note, my default style is a bit more nuanced than inside or outside---I put the period inside the final quote if the whole sentence is inside the quote, basically the same rules used for parenthesis but applied to quotes.  I call this "programmer quotes" because it usually pleases programmers to have operators like sentence-start-capital, sentence-end-period, start-quote, end-quote in properly nested order.

Reply:
@jonatack: From what I read in American English commas and periods are placed inside of the quotation marks, British English places them depending on whether they belong to the quoted sequence or the carrier sentence—essentially what @harding described as "programmer quotes".

Reply:
Marking resolved.